guides/common/modules/proc_importing-kickstart-repositories.adoc

```diff
@@ -1,107 +1,101 @@
 [id="Importing_Kickstart_Repositories_{context}"]
-= Importing Kickstart Repositories for {RHEL}{nbsp}{rhel-version}
+= Importing Kickstart Repositories for {RHEL}{nbsp}{el-major-version}
 
-To import Kickstart repositories for {RHEL}{nbsp}{rhel-version}, complete the following steps on {Project}.
+Use this procedure to import Kickstart repositories for {RHEL}{nbsp}{el-major-version}.
 
 .Procedure
 . Navigate to the Red{nbsp}Hat Customer Portal at https://access.redhat.com/[access.redhat.com] and log in.
-. In the upper left of the window, click *Downloads*.
-
-ifeval::["{rhel-version}" == "8"]
-. Click  {RHEL} 8*.
-. In the Download {RHEL} window, locate the binary DVD version of the ISO image, for example, *{RHEL} 8.1 Binary DVD*, and click *Download Now*.
-endif::[]
-
-ifeval::["{rhel-version}" == "7"]
-. To the right of *{RHEL} 7*, click *Versions 7 and below*.
-. From the *Version* list, select the required version of the {RHEL} 7, for example 7.7.
-. In the Download {RHEL} window, locate the binary DVD version of the ISO image, for example, *{RHEL} 7.7 Binary DVD*, and click *Download Now*.
+. In the upper left corner of the window, click *Downloads*.
```

Comment:
Any reason not to link to https://access.redhat.com/downloads/ instead in the previous step?

Reply:
Good Idea, I have not thought of that.

Reply:
Added.